jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/ContainerizingModeTest.java

```diff
@@ -26,6 +26,7 @@
   public void testFrom_validValues() throws InvalidContainerizingModeException {
     Assert.assertEquals(ContainerizingMode.EXPLODED, ContainerizingMode.from("exploded"));
     Assert.assertEquals(ContainerizingMode.PACKAGED, ContainerizingMode.from("packaged"));
+    Assert.assertEquals(ContainerizingMode.NATIVE_IMAGE, ContainerizingMode.from("native_image"));
```

Comment:
should `native_image` -> `nativeImage`?

Reply:
ContainerizingMode currently has a strong requirement that it all be in lower-case:
https://github.com/GoogleContainerTools/jib/blob/4f438d4c66775a34f433eacd5e3758d69ac00d03/jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/ContainerizingMode.java#L42-L47

Reply:
Oh so this is an issue directly tied to enums, which means `native-image` would also be invalid? It just feels weird to use `_` when we don't use it anywhere else.

Reply:
but I guess this will be very explicitly experimental so we can change it whenever.